app/models/response_template.rb

```diff
@@ -14,7 +13,7 @@ class ResponseTemplate < ApplicationRecord
   USER_NIL_TYPE_OF_MSG = "cannot have a user ID associated.".freeze
 
   validates :type_of, :content_type, :content, :title, presence: true
-  validates :content, uniqueness: { scope: UNIQUENESS_SCOPE }
+  validates :content, uniqueness: { scope: %i[user_id type_of content_type] }
```

Comment:
this is because Rubocops unique index static checker needs the columns to be defined inline to be able to validate composite indexes